onnxruntime/python/tools/kernel_explorer/kernels/fast_gelu_test.py

```diff
@@ -96,9 +96,7 @@ def profile_with_args(batch_size, seq_len, hidden_size, dtype):
 def profile():
     for dtype in dtypes:
         for bert_size in get_bert_sizes():
-            for func in dtype_to_funcs(dtype):
-                profile_fast_gelu_func(*bert_size, dtype, func)
-            print()
+            profile_with_args(*bert_size, dtype)
```

Comment:
maybe the `print()` is still needed.

Reply:
also move `print()` to `profile_with_args` function.